lib/jsonapi/utils/support/pagination.rb

```diff
@@ -32,7 +32,7 @@ def pagination_params(records, options)
         end
 
         def paginator(params)
-          page_params = ActionController::Parameters.new(params[:page])
+          page_params = ActionController::Parameters.new(params[:page] || {})
```

Comment:
In Rails 5 ActionController Parameters initialized with nil value fails so we need to give it an empty hash in that case.